MTA-4505 Release Notes and Attributes Update for 7.2.2 (#135)
migtools/mta-documentation: docs/topics/ref_known-issues-7-2-2.adoc

:_template-generated: 2024-12-04
:_mod-docs-content-type: REFERENCE

[id="known-issues-7-2-2_{context}"]
= Known issues

This section provides highlighted known issues in {ProductFullName} version 7.2.2.

.MTA generates false positive issues in Java application analysis

> When you analyze Java applications, {ProductShortName} rules trigger a false positive issue by matching the `line 1` of a Java file. Currently, there is no workaround for this issue. link:https://issues.redhat.com/browse/MTA-4918[(MTA-4918)]

Reviewer: Might be important to note that this _can_ occur. Most of the time, it does not. Some line 1 matches will still be valid.

Author: Thanks, @eemcmullan. I modified the sentence to include 'can' that conveys your explanation better.

> .Source code and dependency analyses of Java applications generate varying effort

Reviewer 2: ```suggestion
.Source code and dependency analyzes of Java applications generate varying effort
```

> .Source code and dependency analyses of Java applications generate varying effort

Reviewer 2: ```suggestion
.Source code and dependency analysis of Java applications generates varying effort
```

> When you run multiple `source code + dependency` analyses of Java applications for the `OpenJDK 17` target, {ProductShortName} generates varying effort. Currently, there is no workaround for this issue. link:https://issues.redhat.com/browse/MTA-4943[(MTA-4943)]

Reviewer 2: ```suggestion
When you run multiple `source code + dependency` analyzes of Java applications for the `OpenJDK 17` target, {ProductShortName} generates varying effort. Currently, there is no workaround for this issue. link:https://issues.redhat.com/browse/MTA-4943[(MTA-4943)]
```

Author: Thanks! I retained 'analyses' since it takes multiple analyses to produce varying effort, wherever applicable in the doc.
